runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -226,8 +237,10 @@ private JPanel buildGroupPanel()
 	{
 		// Create base panel for the config button and enabled/disabled button
 		final JPanel groupPanel = new JPanel();
+		groupPanel.setBorder(new EmptyBorder(0, 10, 0, 10));
 		groupPanel.setLayout(new BorderLayout(3, 0));
-		groupPanel.setPreferredSize(new Dimension(PluginPanel.PANEL_WIDTH, 20));
+		groupPanel.setPreferredSize(CONFIG_ROW_DIMENSIONS);
+		groupPanel.setOpaque(false);
```

Comment:
I'm pretty sure this would re-introduce the performance issues fixed in #2964

Reply:
Ah, thanks for pointing me to this. Psikoi probably knows the UI code better than any of us now, so I'll follow his advice here.